spec/classes/puppet_agent_windows_install_spec.rb

```diff
@@ -222,6 +222,35 @@
           }
         end
       end
+    end 
```

Comment:
This end is extra, removing it will make the tests pass:
```diff
❯ git diff
----------------------------------------------------------------------------------------
modified: spec/classes/puppet_agent_windows_install_spec.rb
----------------------------------------------------------------------------------------
@@ -222,7 +222,6 @@ RSpec.describe 'puppet_agent', tag: 'win' do
           }
         end
       end
-    end

       context 'wait_for_pxp_agent_exit =>' do
         describe 'default' do
@@ -292,4 +291,4 @@ RSpec.describe 'puppet_agent', tag: 'win' do
       end
     end
   end
-end
\ No newline at end of file
+end

```

```
Finished in 1 minute 29.4 seconds (files took 1 minute 52.77 seconds to load)
1863 examples, 0 failures
```